apps/heureka/src/components/Services/utils.ts

```diff
@@ -205,14 +217,14 @@ export type Issue = {
 
 type NormalizedImageVersionIssues = {
   issues: Issue[]
-  totalCount: number
+  total: number
```

Comment:
same as above, may be we can change it to `totalNumberOfImageVersionIssues` or just `totalImageVersionIssues`.